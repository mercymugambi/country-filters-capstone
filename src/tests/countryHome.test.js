import { configureStore } from '@reduxjs/toolkit';
import CountryReducer from '../redux/region/CountrySlice';

describe('Store Configuration', () => {
  test('return the correct reducer', () => {
    const store = configureStore({
      reducer: {
        countries: CountryReducer,
      },
    });

    expect(store.getState().countries).toMatchSnapshot();
  });
});
