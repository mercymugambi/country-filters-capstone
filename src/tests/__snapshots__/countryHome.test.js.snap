// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store Configuration return the correct reducer 1`] = `
Object {
  "countries": Array [],
  "loading": false,
}
`;

exports[`Store Configuration should have the correct reducer 1`] = `
Object {
  "countries": Array [],
  "loading": false,
}
`;
